src/decoder.ts

```diff
@@ -52,12 +52,12 @@ export const decoderErrorString = (error: DecoderError): string =>
 /*
  * Helpers
  */
-const isJsonArray = (json: any): boolean => json instanceof Array;
+const isJsonArray = (json: any): json is unknown[] => json instanceof Array;
```

Comment:
I just checked and it looks like `Array.isArray` is universally supported [without the need for a dep or polyfill](http://kangax.github.io/compat-table/es5/#test-Array.isArray). I think it'd be a worthwhile and safe change to make.

Reply:
Please do, that would be great!